src/Build/BackEnd/Shared/BuildResult.cs

```diff
@@ -33,7 +33,7 @@ public enum BuildResultCode
     /// Contains the current results for all of the targets which have produced results for a particular configuration.
     /// </summary>
     /// <remarks>
-    /// When modifying serialization/deserialization, bump the version and support previous versions in order to keep <see cref="ResultsCache"/> backwards compatible.
+    /// When modifying serialization/deserialization, bump the version and support previous versions in order to keep results cache backwards compatible.
     /// </remarks>
```

Comment:
I'd move this outside of a `<remarks>`; it's not a user-facing thing but a note for MSBuild maintainers.

Reply:
Done.